documentation/bolt_installing_modules.md

```diff
@@ -1,14 +1,51 @@
-# Modules
-
-Tasks and plans are packaged in Puppet modules, so you can install them as you would any module and manage them with a Puppetfile. 
+# Installing modules
+
+To share the Bolt plans and tasks that you've created on the Forge, you can
+package them into Puppet modules. If you'd like to use a plan or task from a
+module that you found on the Forge, you can use Bolt to install the module.
+
+## Install a module
+
+> **Before you begin**
+> 
+> - In your Bolt project directory, create a file named `Puppetfile`. 
+> - Add any task or plan modules stored locally in `modules/` to the list. For
+>   example, 
+>   ```puppet
+>     mod 'my_awesome_module', local: true
+>   ```
+> 
+>   **Bolt deletes any content in `modules/` that is not listed in your
+>   Puppetfile.** If you
+>   want to keep the content, but you don't want to manage it with the Puppetfile,
+>   move the content to a `site-modules` directory in your project.
```

Comment:
I believe all of the content in `modules/` will be deleted before installing the modules in the Puppetfile. So even if `modules/` contained only the modules in the Puppetfile, any changes you made to those modules will be deleted when running `bolt puppetfile install`.

Reply:
I actually checked this (cause I thought the same) and it's not! If you modify a module that's in the Puppetfile, if doesn't look to get deleted.

Reply:
That's weird. I also checked and it removes any changes I made, whether it's new files/directories or changes to existing files.

Reply:
Huh, it might just be the `facts` module that that applies to...